src/rating/Rating.js

```diff
@@ -69,28 +69,40 @@ export default class Rating extends Component {
         this.setState({ value: gesture.dx });
       },
       onPanResponderRelease: event => {
-        onFinishRating(this.getCurrentRating());
+        const rating = this.getCurrentRating();
+        if (!fractions) {
+          // "round up" to the nearest star/rocket/whatever
+          this.setCurrentRating(rating);
```

Comment:
> individual images will "fill up" to match the rounding up of rating values.

Can you explain this in a  bit more detail? Adding a couple screenshots for a few fractions would be great.